### PR TITLE
[tensorflow] [trcomp] [huggingface] add trcomp 2.6 dockerfile with latest CVE fixes

### PR DESCRIPTION
**Pull request by codeislife99**
*GitHub Issue #, if available:*

Note: If merging this PR should also close the associated Issue, please also add that Issue # to the Linked Issues section on the right. 

### Tests run
**NOTE: If you are creating a PR for a new framework version, please ensure success of the standard, rc, and efa sagemaker remote tests by updating the dlc_developer_config.toml file:**

- [ ] Revision A: `sagemaker_remote_tests = "standard"`
- [ ] Revision B: `sagemaker_remote_tests = "rc"`
- [ ] Revision C: `sagemaker_remote_tests = "efa"`

**Additionally, please run the sagemaker local tests in at least one revision:**
- [ ] `sagemaker_local_tests = true`

## Label Checklist
- [ ] I have added the project label for this PR (*<project_name>* or "Improvement")

## PR Checklist
- [ ] I've prepended PR tag with frameworks/job this applies to : [mxnet, tensorflow, pytorch] | [ei/neuron/graviton] | [build] | [test] | [benchmark] | [ec2, ecs, eks, sagemaker]
- [ ] If the PR changes affects SM test, I've modified dlc_developer_config.toml in my PR branch by setting sagemaker_tests = true and efa_tests = true
- [ ] If this PR changes existing code, the change fully backward compatible with pre-existing code. (Non backward-compatible changes need special approval.)
- [ ] (If applicable) I've documented below the DLC image/dockerfile this relates to
- [ ] (If applicable) I've documented below the tests I've run on the DLC image
- [ ] (If applicable) I've reviewed the licenses of updated and new binaries and their dependencies to make sure all licenses are on the Apache Software Foundation Third Party License Policy Category A or Category B license list.  See [https://www.apache.org/legal/resolved.html](https://www.apache.org/legal/resolved.html).
- [ ] (If applicable) I've scanned the updated and new binaries to make sure they do not have vulnerabilities associated with them.

## Pytest Marker Checklist
- [ ] (If applicable) I have added the marker `@pytest.mark.model("<model-type>")` to the new tests which I have added, to specify the Deep Learning model that is used in the test (use `"N/A"` if the test doesn't use a model)
- [ ] (If applicable) I have added the marker `@pytest.mark.integration("<feature-being-tested>")` to the new tests which I have added, to specify the feature that will be tested
- [ ] (If applicable) I have added the marker `@pytest.mark.multinode(<integer-num-nodes>)` to the new tests which I have added, to specify the number of nodes used on a multi-node test
- [ ] (If applicable) I have added the marker `@pytest.mark.processor(<"cpu"/"gpu"/"eia"/"neuron">)` to the new tests which I have added, if a test is specifically applicable to only one processor type

#### EIA/NEURON/GRAVITON Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `ei_mode = true`, `neuron_mode = true` or `graviton_mode = true`

#### Benchmark Testing Checklist
* When creating a PR:
- [ ] I've modified `dlc_developer_config.toml` in my PR branch by setting `benchmark_mode = true`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license. I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Comment by Lokiiiiii** [comment]
@tejaschumbalkar  Could you please check why the tests are blocked ?

**Comment by tejaschumbalkar** [comment]
> @tejaschumbalkar Could you please check why the tests are blocked ?

Please add @codeislife99 to dlc-trcomp-reviewers

**Comment by codeislife99** [comment]
The CI is failing because the numpy version and some other things are not changed in the base image.

**Comment by codeislife99** [comment]
The CI seems to be failing because of Opencv & numpy vulnerabilities in the base image. THey are still 4.5.5 and 1.19 respectively.

File: huggingface/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.trcomp.gpu [hunk 1]
```diff
@@ -29,8 +29,6 @@ RUN pip install --no-cache-dir \
     tensorflow-addons==0.12.0 \
     psutil \
     "typing-extensions==3.7.4.3" \
```
Comment (Lokiiiiii) [inline huggingface/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.trcomp.gpu:31]:
```suggestion
    "typing-extensions==3.7.4.3"
```

**Review by Lokiiiiii — CHANGES_REQUESTED** [review]

**Review by Lokiiiiii — DISMISSED** [review]

**Comment by codeislife99** [comment]
@tejaschumbalkar - Many of the tests are failing because of capacity errors and because of vulnerabilities in the base image. What is the recommended solution here ?

**Comment by saimidu** [comment]
@codeislife99 As of now, the only solution is to rerun the tests. All the tests that previously succeeded are not going to be rerun the next time.

File: huggingface/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.trcomp.gpu (hunk 1 above)
Comment (tejaschumbalkar) [reply huggingface/tensorflow/training/docker/2.6/py3/cu112/Dockerfile.trcomp.gpu]: Why is this package pinned?

**Comment by Qingzi-Lan** [comment]
tests passed on commit [065a13d]

**Comment by codeislife99** [comment]
@saimidu @Lokiiiiii Can I get a final thumbs up for merge , all tests have passed in previous commit and I reverted the dlc_developer_config file.

**Review by tejaschumbalkar — APPROVED** [review]

**Comment by tejaschumbalkar** [comment]
@Lokiiiiii Can you review/approve the PR?

**Review by Lokiiiiii — APPROVED** [review]

**Comment by codeislife99** [comment]
Can we merge this PR now ? All tests have already passed in a previous similar commit

**Pull request merged**